app/org/maproulette/models/dal/ChallengeDAL.scala

```diff
@@ -691,6 +691,11 @@ class ChallengeDAL @Inject() (override val db:Database, taskDAL: TaskDAL,
         case _ =>
       }
 
+      searchParameters.challengeDifficulty match {
+        case Some(v) if v != -1 => this.appendInWhereClause(whereClause, s"c.difficulty = ${v}")
```

Comment:
I would say if v > 0 & v < 4, as those are the only valid values for challenge difficulty. (Ie. 1, 2, 3)